### PR TITLE
Seeds - added reviews

### PR DESCRIPTION
**Pull request by qqdipps**

### CODE REVIEW

File: db/reviews_data.csv [hunk 1]
```diff
@@ -0,0 +1,41 @@
+rating,comment
+4,"Bestfriend time brightened my day. It was also promises to make shooting daytime and nighttime action simpler and professional-looking, providing the budding photographer with new and enticing reasons to hone their skills."
+4,"Ada best seller pack brightened my day. It was also promises to make shooting daytime and nighttime action simpler and professional-looking, providing the budding photographer with new and enticing reasons to hone their skills."
+1,Cup of rainbows design does not hold up on rugged use and did not reduce fatique.
```
Comment (dev-elle-up) [inline db/reviews_data.csv:4]: It looks like some of these comments are surrounded by strings and some are not. Does this matter? Just wanna check.

File: db/reviews_generate_seeds.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+require "faker"
+require "csv"
+
+CSV.open("db/reviews_data.csv", "w", :write_headers => true,
+                                     :headers => ["rating", "comment"]) do |csv|
+  intangibles = [
+    "high fives", "cup of sunshine", "bucket of sunshine", "sleep revival", "relaxation", "energy", "a good hair day", "pause time", "freshly showered", "teleport into bed", "ada starter kit", "ada best seller pack", "cup of rainbows", "puppy play pack", "kitten cuddles", "giggle party", "bestfriend time", "juicy secret", "inside joke moment", "bondfire time",
```
Comment (dev-elle-up) [inline db/reviews_generate_seeds.rb:7]: I think you don't need the comma after the last item in an array.

File: db/seeds.rb [hunk 3]
```diff
@@ -26,3 +26,13 @@
                                      stock: line["stock"])
   product.valid? ? (puts "#{i + 1} product") : failed_to_save << product
 end
```
Comment (dev-elle-up) [inline db/seeds.rb:28]: Cool, thanks for adding this!!

File: db/seeds.rb [hunk 4]
```diff
@@ -26,3 +26,13 @@
...
+
+CSV.open("db/reviews_data.csv", headers: true).each_with_index do |line, i|
+  product = Product.find(rand(1..Product.count))
+  review = Review.new(rating: line["rating"],
+                      comment: line["comment"])
+  product.reviews << review
+  review.valid? ? (puts "#{i + 1} review") : failed_to_save << review
+end
+
+puts "#{failed_to_save.count} failed to save"
```
Comment (dev-elle-up) [inline db/seeds.rb:38]: And this! :D

File: review_data.csv [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+2,Giggle party design does not hold up on rugged use and did not reduce fatique. 
+2,Teleport into bed design does not hold up on rugged use and did not reduce fatique. 
+1,Cup of sunshine design does not hold up on rugged use and did not reduce fatique. 
+5,"Bondfire time brightened my day. It was also promises to make shooting daytime and nighttime action simpler and professional-looking, providing the budding photographer with new and enticing reasons to hone their skills."
+2,Ada starter kit design does not hold up on rugged use and did not reduce fatique. 
+2,Juicy secret design does not hold up on rugged use and did not reduce fatique. 
+4,"Bestfriend time brightened my day. It was also promises to make shooting daytime and nighttime action simpler and professional-looking, providing the budding photographer with new and enticing reasons to hone their skills."
+5,"Giggle party brightened my day. It was also promises to make shooting daytime and nighttime action simpler and professional-looking, providing the budding photographer with new and enticing reasons to hone their skills."
+1,Cup of rainbows design does not hold up on rugged use and did not reduce fatique.
```
Comment (dev-elle-up) [inline review_data.csv:41]: Same question about quotes.

**Review by dev-elle-up — CHANGES_REQUESTED** [review]
Looking good. Please address a few questions and re-submit (or whatever needs to happen so I can review changes again before merging).

**Comment by qqdipps** [comment]
I don't think that the "" are an issue, everythings works out okay in the db.

File: review_data.csv (hunk 5 above)
Comment (qqdipps) [reply review_data.csv:41]: Looks ok in db

File: db/reviews_generate_seeds.rb (hunk 2 above)
Comment (qqdipps) [reply db/reviews_generate_seeds.rb:7]: this is an auto thing

File: db/reviews_data.csv (hunk 1 above)
Comment (qqdipps) [reply db/reviews_data.csv:4]: I think its fine with or without, saves good in db.

**Review by dev-elle-up — APPROVED** [review]

**Pull request merged**